scarb/src/ops/lint.rs

```diff
@@ -53,6 +57,14 @@ pub fn lint(opts: LintOptions, ws: &Workspace<'_>) -> Result<()> {
         },
     )?;
 
+    let absolute_path = opts
+        .path
+        .as_ref()
+        .map(|path| {
+            canonicalize(path).map_err(|_| anyhow!("Provided `SCARB_ACTION_PATH` is invalid"))
+        })
+        .transpose()?;
```

Comment:
```suggestion
        .map(canonicalize)
        .transpose()?;
```
Canonicalize will return an error that can be helpful for the user (https://github.com/software-mansion/scarb/blob/4388a3801ef5a7f4a72651b08873abeaa76a2fa4/scarb/src/internal/fsx.rs#L18)

If you need to add additional context, use `.with_context` instead of replacing the error fully. 

Also, I would not phrase it as `provided ENV_NAME`, as it can be passed via cli (mostly will).